### PR TITLE
vision/detect: add web entity geo sample

### PR DESCRIPTION
**Pull request by broady**

### CODE REVIEW

File: vision/detect/detect.go [hunk 1]
```diff
@@ -399,6 +401,49 @@ func detectWeb(w io.Writer, file string) error {
...
+	}
+	web, err := client.DetectWeb(ctx, image, imageContext)
+	if err != nil {
+		return err
+	}
+
+	if len(web.WebEntities) != 0 {
+		fmt.Fprintln(w, "Entities:")
+		for _, entity := range web.WebEntities {
+			fmt.Fprintf(w, "\t%-12s %s\n", entity.EntityId, entity.Description)
```
[the remark this answers is not in the document]
Comment (broady) [reply vision/detect/detect.go]: Yes, it's included for the main "web entity" sample. If it's not wanted, we can remove it from all of the samples.

[the remark this answers is not in the document]
Comment (broady) [reply vision/detect/detect.go]: oh, right, you're saying it's missing! yes, I can add it.

edit: looks like score isn't included in any existing samples. probably best we do that in a separate round of changes. could you file a bug in this repo?

File: vision/detect/detect.go [hunk 2]
```diff
@@ -731,6 +776,40 @@ func detectWebURI(w io.Writer, file string) error {
...
+
+// detectWebGeo detects geographic metadata from the Vision API for an image at the given file path.
+func detectWebGeoURI(w io.Writer, file string) error {
+	ctx := context.Background()
+
+	client, err := vision.NewImageAnnotatorClient(ctx)
+	if err != nil {
+		return err
+	}
+
```
[the remark this answers is not in the document]
Comment (broady) [reply vision/detect/detect.go]: hm, good point. could you file a bug - if this is what we want, we should do this for all of the samples.

File: vision/detect/detect_test.go [hunk 3]
```diff
@@ -33,9 +33,14 @@ func TestDetect(t *testing.T) {
 		{"FullText", detectDocumentText, detectDocumentTextURI, "text.jpg", "Preparing to install"},
 		{"Crop", detectCropHints, detectCropHintsURI, "wakeupcat.jpg", "(0,0)"},
 		{"Web", detectWeb, detectWebURI, "wakeupcat.jpg", "Web properties"},
+		{"WebGeo", nil, detectWebGeoURI, "city.jpg", "Zepra"},
```
Comment (davidcavazos) [inline vision/detect/detect_test.go]: Is there a reason to skip the local detection?

**Pull request merged**